src/GitRocketTerminal.jsx

```diff
@@ -8,9 +8,9 @@ import { Rocket, RocketSpan, FinTop, FinBottom, Fire, Wastes } from './styledEle
 // at the command line. Currently it supports output from bash, zsh, fish, cmd and powershell.
 function detectPushCommand(data) {
   const patterns = [
-    'To(.+)\.git',
+    'To(.+)\.git', 'error:'
   ];
-  return new RegExp(`(${patterns.join(')|(')})`).test(data);
+  return new RegExp(`(${patterns.join(')|(')})`).test(data) && !new RegExp(patterns[1]).test(data);
```

Comment:
In this line the  `${patterns.join(')|(')}))` portion actually combines all of the patterns in the `patterns` array into one. Because of that, this new RegExp is actually looking for things that both match the`To(.+)\.git` pattern and `error:` before you add the second fragment && !new RegExp(patterns[1]).test(data) to ignore lines that match `error:`.

Could we actually create a new array for anti-patterns (one for things we want to match for and things we don't)? Then your additional check could pull from that array. There's probably a way to do this in one normal regex as well, but since this work I'm happy to merge and release!